src/provider/controller/IntentHandler.ts

```diff
@@ -166,3 +168,8 @@ interface IntentWithTarget extends Intent {
 function hasTarget(intent: Intent): intent is IntentWithTarget {
     return !!intent.target;
 }
+
+// Check if the intent handler return value is valid defined.
+function returnValuePredicate(value?: any) {
```

Comment:
Same comment as for this function in client

Reply:
Changed the rules to be any defined value. Removed this predicate function.

Reply:
@tim-dinsdale Gonna assume you're fine for me to resolve this!